samples/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# Samples do not need to be built for NPM package
+if(CPACK_GENERATOR STREQUAL "NPM")
+    return()
```

Comment:
maybe we need to introduce ENABLE_SAMPLES option and turn if off for NPM generator?

and re-use here https://github.com/openvinotoolkit/openvino.genai/blob/7765bc3de68b1ea10916328713796ce08b0e5641/CMakeLists.txt#L91-L93

and pass here https://github.com/openvinotoolkit/openvino.genai/blob/7765bc3de68b1ea10916328713796ce08b0e5641/pyproject.toml#L50

Reply:
Do you mean like this: https://github.com/openvinotoolkit/openvino.genai/pull/1193/commits/c8eee1e6f6addddd4cd3d8857dff69feaab96b1f ?